sdk/storage/storage-file-datalake/review/storage-file-datalake.api.md

```diff
@@ -0,0 +1,1188 @@
+  // (undocumented)
+  getAccessControl(options?: PathGetAccessControlOptions): Promise<PathGetAccessControlResponse>;
+  // (undocumented)
+  getDataLakeLeaseClient(proposeLeaseId?: string): DataLakeLeaseClient;
+  // (undocumented)
+  getProperties(options?: PathGetPropertiesOptions): Promise<PathGetPropertiesResponse>;
+  // (undocumented)
+  move(destinationPath: string, options?: PathMoveOptions): Promise<PathMoveResponse>;
+  // (undocumented)
+  readonly name: string;
+  // (undocumented)
+  setAccessControl(
+    acl: PathAccessControlItem[],
+    options?: PathSetAccessControlOptions
+  ): Promise<PathSetAccessControlResponse>;
+  // (undocumented)
+  setHttpHeaders(
+    httpHeaders: PathHttpHeaders,
+    options?: PathSetHttpHeadersOptions
+  ): Promise<PathSetHttpHeadersResponse>;
+  // (undocumented)
+  setMetadata(
+    metadata: Metadata,
+    options?: PathSetMetadataOptions
+  ): Promise<PathSetMetadataResponse>;
+  // (undocumented)
+  setPermissions(
+    permissions: PathPermissions,
+    options?: PathSetPermissionsOptions
+  ): Promise<PathSetAccessControlResponse>;
+  // (undocumented)
+  toDirectoryClient(): DataLakeDirectoryClient;
+  // (undocumented)
+  toFileClient(): DataLakeFileClient;
+}
+
+// @public (undocumented)
+export interface DataLakeRequestConditions
+  extends ModifiedAccessConditions,
+    LeaseAccessConditions {}
+
+// @public (undocumented)
+export class DataLakeServiceClient extends StorageClient {
+  constructor(
+    url: string,
+    credential?: StorageSharedKeyCredential | AnonymousCredential | TokenCredential,
+    options?: StoragePipelineOptions
+  );
+  constructor(url: string, pipeline: Pipeline);
+  // (undocumented)
+  getFileSystemClient(fileSystemName: string): DataLakeFileSystemClient;
+  // (undocumented)
+  getUserDelegationKey(
+    startsOn: Date,
+    expiresOn: Date,
+    options?: ServiceGetUserDelegationKeyOptions
+  ): Promise<ServiceGetUserDelegationKeyResponse>;
+  // (undocumented)
+  listFileSystems(
+    options?: ServiceListFileSystemsOptions
+  ): PagedAsyncIterableIterator<FileSystemItem, ServiceListFileSystemsSegmentResponse>;
+}
+
+export { deserializationPolicy };
+
+// @public (undocumented)
+export interface FileAppendOptions {
+  // (undocumented)
+  abortSignal?: AbortSignalLike_2;
+  // (undocumented)
+  conditions?: LeaseAccessConditions;
+  // (undocumented)
+  contentHash?: Uint8Array;
+}
+
+// @public (undocumented)
+export interface FileFlushOptions {
+  // (undocumented)
+  abortSignal?: AbortSignalLike_2;
+  // (undocumented)
+  close?: boolean;
+  // (undocumented)
+  conditions?: DataLakeRequestConditions;
+  // (undocumented)
+  pathHttpHeaders?: PathHttpHeaders;
+  // (undocumented)
+  retainUncommittedData?: boolean;
+}
+
+// @public (undocumented)
+export interface FileReadHeaders {
+  // (undocumented)
+  acceptRanges?: string;
+  // (undocumented)
+  blobCommittedBlockCount?: number;
+  // (undocumented)
+  blobContentMD5?: Uint8Array;
```

Comment:
`blobContentMD5` and several below - should this have a different prefix? We currently have `blob-` for storage-blob and `file` for storage-file-share.

Reply:
Updated.

Reply:
Renamed `blobContentMD5` to `fileContentMD5`